symposion/templates/reviews/_review_table.html

```diff
@@ -99,15 +99,36 @@
     </tbody>
 </table>
 
+<!-- To change the status of multiple proposals at the same time -->
+<form action="" method="POST">
+  {% csrf_token %}
+  <button id="button" type="submit" class="btn btn-primary">
+    <i class="icon-check"></i>
+    Change status of selected rows to
+  </button>
+
+<!-- We use jQuery below to change the value of this input to the selected rows -->
+  <input id="multiInput" type="hidden" name="pk" value="" />
+  <select name="status">
+    {% for k, v in status_options %}
+      <option value="{{ k }}" {% if proposal.overall_status == k %}selected="selected"{% endif %}>
+        {{ v }}
+      </option>
+    {% endfor %}
+  </select>
+</form>
+
+
 {% compress js %}
-<script src="{{ STATIC_URL }}datatables/js/jquery.dataTables.min.js" type="text/javascript"></script>
+<script src="{{ STATIC_URL }}datatables/js/jquery-1.11.1.min.js" type="text/javascript"></script>
+<script src="{{ STATIC_URL }}datatables/js/jquery.dataTables-1.10.7.js" type="text/javascript"></script>
```

Comment:
It seems like something must have been loading datatables already, since this page worked before (though I haven't tried to figure out where).